backend/app/resources/api/v1/transportation_analysis_resource.rb

```diff
@@ -2,8 +2,13 @@ class Api::V1::TransportationAnalysisResource < JSONAPI::Resource
   attributes(
     :traffic_zone,
     :jtw_study_selection,
-    :required_jtw_study_selection
+    :required_jtw_study_selection,
+    :jtw_study_area_lat_lon
```

Comment:
just returning the native `jtw_study_area_centroid` results in a string like : ```POINT(X Y)``` being stored by ember. We mostly want to use this as a LatLng-like array of points, so I'm returning it like that. If it turns out there are other ways we want to use this centroid, and it would be better represented as some other form then can easily update the resource.